workspaces/bulk-import/plugins/bulk-import-backend/src/schema/openapi.yaml

```diff
@@ -570,6 +570,25 @@ components:
                   type: string
                   description: content of the catalog-info.yaml as fetched from the Pull Request.
 
+    SourceImport:
+      title: Import Job with source it originates from
+      allOf:
+        - $ref: '#/components/schemas/Import'
+        - type: object
+          properties:
+            source:
+              $ref: '#/components/schemas/Source'
+
+    Source:
+      type: string
+      nullable: true
+      description: Import Source
```

Comment:
Could you quickly expand this description by documenting what each enum value means?